bin/analytics

```diff
@@ -10,143 +10,82 @@ program :name, 'simulator.rb'
 program :version, '0.0.1'
 program :description, 'scripting simulator'
 
-# use an env var for write key, instead of a flag
-Analytics = Segment::Analytics.new({
-  write_key: ENV['SEGMENT_WRITE_KEY'],
-  on_error: Proc.new { |status, msg| print msg }
-})
-
 def toObject(str)
```

Comment:
I'd suggest renaming this to `to_hash`. `toObject` looks like it was copied from a JS implementation..

Reply:
I couldn't get `to_hash` to work for some reason. Possibly related to https://github.com/Shopify/ssllabs.rb/pull/31/files ? Instead I named it `json_hash`, but open to other suggestions.

```
→ go run cmd/cli/main.go -segment-write-key='OnMMoZ6YVozrgSBeZ9FpkC0ixH0ycYZn' -runscope-token='92f6b74c-0200-4735-9195-702d528aa97f' -runscope-bucket='bmbbsjxdttr7' -path='../analytics-ruby/bin/analytics'
../analytics-ruby/bin/analytics:13:in `to_hash': wrong number of arguments (given 0, expected 1) (ArgumentError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:408:in `gsub'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:408:in `regexp'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:412:in `candidate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:429:in `candidate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:434:in `complete'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:839:in `complete'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1701:in `block in visit'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1700:in `reverse_each'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1700:in `visit'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1732:in `block in complete'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1731:in `catch'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1731:in `complete'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1541:in `block in parse_in_order'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1534:in `catch'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1534:in `parse_in_order'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1528:in `order!'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1620:in `permute!'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/optparse.rb:1642:in `parse!'
	from /Library/Ruby/Gems/2.3.0/gems/commander-4.4.4/lib/commander/runner.rb:381:in `parse_global_options'
	from /Library/Ruby/Gems/2.3.0/gems/commander-4.4.4/lib/commander/runner.rb:65:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/commander-4.4.4/lib/commander/delegates.rb:15:in `run!'
	from /Library/Ruby/Gems/2.3.0/gems/commander-4.4.4/lib/commander/import.rb:5:in `block in <top (required)>'
main[17633]: 2018-03-06 15:26:50.003 - cli/main.go:38 - test error: could not invoke command: exit status 1
exit status 1

```